casepro/backend/junebug.py

```diff
@@ -357,7 +366,22 @@ def push_outgoing(self, org, outgoing, as_broadcast=False):
         :param as_broadcast: whether outgoing messages differ only by recipient and so can be sent as single broadcast
         """
         for message in outgoing:
-            self.message_sender.send_message(message)
+            self.send_message(message)
+
+    def send_message(self, outgoing):
+        message = outgoing.reply_to
```

Comment:
Yeah, agreed but I would suggest we do that in a separate PR as it's essentially a different feature and would likely also require UI work to expose that.